pykokkos/interface/views.py

```diff
@@ -532,6 +532,10 @@ def asarray(obj, /, *, dtype=None, device=None, copy=None):
     # TODO: proper implementation/design
     # for now, let's cheat and use NumPy asarray() followed
     # by pykokkos from_numpy()
+    if obj in [pk.e, pk.pi, pk.inf, pk.nan]:
```

Comment:
Minor comment, but isn't it better to use a `set` here? So `{pk.e, pk.pi, pk.inf, pk.nan}`